.github/workflows/helm-validation.yml

```diff
@@ -0,0 +1,47 @@
+name: Helm validation
+
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Validators setup
+        env:
+          KUBESCORE_VERSON: 1.7.1
+        run: |
+          mkdir /tmp/k8s-validators
+          cd /tmp/k8s-validators
+
+          curl -LO https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz
+          tar xf kubeval-linux-amd64.tar.gz
+          rm -f kubeval-linux-amd64.tar.gz LICENSE README.md
+
+          curl -LO https://github.com/zegl/kube-score/releases/download/v${{ env.KUBESCORE_VERSON }}/kube-score_${{ env.KUBESCORE_VERSON }}_linux_amd64.tar.gz
+          tar xf kube-score_${{ env.KUBESCORE_VERSON }}_linux_amd64.tar.gz
+          rm -f kube-score_${{ env.KUBESCORE_VERSON }}_linux_amd64.tar.gz LICENSE README.md
+
+          curl -LO https://get.helm.sh/helm-v3.2.4-linux-amd64.tar.gz
+          tar xf helm-v3.2.4-linux-amd64.tar.gz
+          mv linux-amd64/helm .
+          rm -fr helm-v3.2.4-linux-amd64.tar.gz linux-amd64
+      - name: Cache validators
+        uses: actions/cache@v1
+        with:
+          path: /tmp/k8s-validators
+          key: ${{ runner.os }}-k8s-validators
+      - name: Test freeradius helm chart
+        working-directory: freeradius/helm
+        run: |
+          echo "Kubeval test"
+          helm template -f values.yaml . | /tmp/k8s-validators/kubeval --ignore-missing-schemas
+
+          echo "Kube-score test"
+          helm template -f values.yaml . | /tmp/k8s-validators/kube-score score - || true
```

Comment:
Think we should remove the `|| true`. I just added it for the cloud controller chart cause we agreed on ignoring kube-score results for some time. Don't think we should do the same here.

Reply:
I'm not sure if we really want to do that. Based on https://github.com/Telecominfraproject/wlan-test-bss/runs/2674705279?check_suite_focus=true#step:5:16 , probably testing would simply fail all the time as kube-score is quite strict and I don't agree with everything it considers critical error

Reply:
In this case we can remove it completely from the pipeline.

Reply:
Deleted kube-score and added Helm lint instead